cmd/kubernetes_version.go

```diff
@@ -0,0 +1,155 @@
+package cmd
+
+import (
+	"crypto/tls"
+	"encoding/json"
+	"encoding/pem"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"strings"
+
+	"github.com/golang/glog"
+)
+
+func getKubeVersionFromRESTAPI() (string, error) {
+	k8sVersionURL := getKubernetesURL()
+	serviceaccount := "/var/run/secrets/kubernetes.io/serviceaccount"
+	cacertfile := fmt.Sprintf("%s/ca.crt", serviceaccount)
+	tokenfile := fmt.Sprintf("%s/token", serviceaccount)
+
+	tlsCert, err := loadCertficate(cacertfile)
+	if err != nil {
+		return "", err
+	}
+
+	tb, err := ioutil.ReadFile(tokenfile)
+	if err != nil {
+		return "", err
+	}
+	token := strings.TrimSpace(string(tb))
+
+	data, err := getWebData(k8sVersionURL, token, tlsCert)
+	if err != nil {
+		return "", err
+	}
+
+	k8sVersion, err := extractVersion(data)
+	if err != nil {
+		return "", err
+	}
+	return k8sVersion, nil
+}
+
+func extractVersion(data []byte) (string, error) {
+	/*
+		{
+		  "major": "1",
+		  "minor": "15",
+		  "gitVersion": "v1.15.3",
+		  "gitCommit": "2d3c76f9091b6bec110a5e63777c332469e0cba2",
+		  "gitTreeState": "clean",
+		  "buildDate": "2019-08-20T18:57:36Z",
+		  "goVersion": "go1.12.9",
+		  "compiler": "gc",
+		  "platform": "linux/amd64"
+		}
+	*/
```

Comment:
I think you already added a test for this to document what's proper JSON. We can remove this.